i18n/translations/translations.en.json

```diff
@@ -452,6 +452,12 @@
     "translation": "The census survey is short and on average takes about 10 minutes or less to complete, depending on the number of people in the household."
   },
   "components.FAQ.entries.otherLanguages.question": {
+    "english": "Will the census be available in other languages?",
+    "translation": "Will the census be available in other languages?"
+  },
+  "components.FAQ.entries.otherLanguages.answer": {
+    "english": "Yes. The online survey is available in 12 languages (Spanish, Chinese, Vietnamese, Korean, Russian, Arabic, Tagalog, Polish, French, Haitian Creole, Portuguese, Japanese); phone assistance will also be available in these languages by calling (####). The paper survey is available in English and Spanish. Call to request a paper survey./n The U.S. Census Bureau will provide other materials such as languages guides, glossaries, etc. in 59 languages. Residents who need assistance or who do not have computer or internet access can visit a City library or community center for assistance.",
+    "translation": "Yes. The online survey is available in 12 languages (Spanish, Chinese, Vietnamese, Korean, Russian, Arabic, Tagalog, Polish, French, Haitian Creole, Portuguese, Japanese); phone assistance will also be available in these languages by calling (####). The paper survey is available in English and Spanish. Call to request a paper survey./n The U.S. Census Bureau will provide other materials such as languages guides, glossaries, etc. in 59 languages. Residents who need assistance or who do not have computer or internet access can visit a City library or community center for assistance."
     "english": "I don’t speak English. Will the Census survey be available in other languages?",
```

Comment:
Results in invalid JSON--possibly a merge conflict resolution error?

Reply:
thanks. fixed json errors